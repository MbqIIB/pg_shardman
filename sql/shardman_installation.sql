\set VERBOSITY terse

CREATE EXTENSION pg_shardman CASCADE;
DROP EXTENSION pg_shardman CASCADE;
