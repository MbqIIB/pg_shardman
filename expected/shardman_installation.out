\set VERBOSITY terse
CREATE EXTENSION pg_shardman CASCADE;
NOTICE:  installing required extension "pg_pathman"
NOTICE:  installing required extension "postgres_fdw"
DROP EXTENSION pg_shardman CASCADE;
